src/Polytopes/PolyhedralFan/properties.jl

```diff
@@ -76,14 +76,42 @@ function Base.iterate(iter::MaximalConeIterator, index = 1)
 end
 Base.length(iter::MaximalConeIterator) = nmaximal_cones(iter.PF)
 
-# function cones(as::Type{T}, PF::PolyhedralFan, cone_dim::Int) where {T}
-#     rtype = AsTypeIdentitiesC(as)
-#     if (cone_dim < 0)
-#         return nothing
-#     end
-#     rcones = Polymake.fan.cones_of_dim(PF.pm_fan,cone_dim-length(lineality_space(PF)))
-#     return PolyhedronOrConeIterator{rtype}(PF.pm_fan.RAYS,rcones, PF.pm_fan.LINEALITY_SPACE)
-# end
+@doc Markdown.doc"""
+    cones(as::Type{T} = Cone, PF::PolyhedralFan, cone_dim::Int)
```

Comment:
Maybe this `as` could also take `IncidenceMatrix`. But maybe we should just make this a separate issue.

Reply:
I would prefer a separate issue. I think directly using `IncidenceMatrix` for `cones` would be ambiguous since we can represent the cones via ray- or facet-indices.

Reply:
I pushed some suggestions regarding this on #780.